include/exadg/incompressible_flow_with_transport/driver.cpp

```diff
@@ -507,7 +507,9 @@ Driver<dim, Number>::ale_update() const
   dealii::Timer sub_timer;
 
   sub_timer.restart();
-  grid_motion->update(fluid_time_integrator->get_next_time(), false);
+  grid_motion->update(fluid_time_integrator->get_next_time(),
```

Comment:
similar to the other drivers, we give the actual time step number (in case one uses a `GridMotionElasticity` replacing the currently used `GridMotionFunction` or `GridMotionPoisson`), see `Driver::setup()`. For the currently possible `GridMotion` objects, this would not result in an update. Alternatively we would pass `dealii::numbers::invalid_unsigned_int` or simply 0 in the drivers for iNS and ConvDiff.